WBooks/Screens/BaseViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  BaseViewController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 22/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class BaseViewController: UIViewController {
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        .lightContent
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setNavigationBar()
+        UINavigationBar.appearance().titleTextAttributes = [NSAttributedStringKey.foregroundColor:UIColor.white]
```

Comment:
Is this necessary if you have already set the tintColor in AppDelegate?

Reply:
I think not but for some reason the tint color does not apply

Reply:
applying the Gaby's changes i not need anymore. By the way, tint color is not applying white on title . Just on icons